File: 217-ContainsDuplicate/217-ContainsDuplicate.java
// Last updated: 8/13/2025, 8:21:19 PM
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> hs = new HashSet<>();
        for(int i=0;i<nums.length;i++){
            if(hs.contains(nums[i])==false){
                hs.add(nums[i]);
            }
            else{
                return true;
            }
        }
        return false;
    }
}

File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 8/13/2025, 8:21:09 PM
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        mySet = set()

        for i in range (0,len(nums)):
            if nums[i] in mySet :
                return True
            else :
                mySet.add(nums[i])

        return False
        